response_operations_ui/views/reporting_units.py

```diff
@@ -213,12 +213,26 @@ def view_reporting_unit_survey(ru_ref, survey_id):
     if case is not None and iac_controller.is_iac_active(case["iac"]):
         unused_iac = case["iac"]
 
-    logger.info("Successfully gathered data to view reporting unit survey data", ru_ref=ru_ref, survey_id=survey_id)
-
     permissions = {
         "reporting_unit_edit": user_has_permission("reportingunits.edit"),
         "messages_edit": user_has_permission("messages.edit"),
     }
+
+    enrolment_code_hyperlink = ""
```

Comment:
No point delegating this to a method in the method in `contexts/reporting_units.py` as the signature would be the same amount of code and there's no other use cases for this link.